README.md

```diff
@@ -1,10 +1,10 @@
-# Bridg-It
```

Comment:
Aici n-ai nici o modificare, totusi pare ca ai modificat tot, nu ti se pare ciudat? E din vina line ending-urilor. Tu esti pe windows, le pune CRLF. Nu-i okay.
Ca sa rezolvam, baga comenzile astea:

git config --global core.autocrlf false
git config --global core.eol lf
apoi in root-ul folderului BridgIt, fa un fisier .gitattributes si baga asta:
* text=auto 
in el.

Apoi, ca sa-ti normalizeze line ending-urile, scrie asta in consola de git:

git rm --cached -rf .
git diff --cached --name-only -z | xargs -n 50 -0 git add -f